test/models/user_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
```

Comment:
Your test is incomplete as it is. What we wanna see in the tests are:

1. Testing for valid user creation
2. Testing for invalid user creation
3. Testing all associations of user